integreat_cms/cms/forms/languages/language_form.py

```diff
@@ -54,6 +56,26 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.fields["primary_country_code"].choices = sorted_language_choices
         self.fields["secondary_country_code"].choices = sorted_language_choices
 
+        # Make the language color field required
+        self.fields["language_color"].required = True
+
+        # Show unused colors first in the list, after them used colors with the assigned language name
+        used_colors = []
+        unused_colors = []
+        for color in language_color.COLORS:
+            code, name = color
```

Comment:
```suggestion
            color_code, name = color
```
I'd rather name this color_code to be more specific, but I'll leave it up to you to decide :)